pkg/cmd/cli/describe/projectstatus.go

```diff
@@ -489,11 +492,124 @@ func describePodInServiceGroup(f formatter, podNode *kubegraph.PodNode) []string
 	return lines
 }
 
+type exposedRoutes []string
+
+func (e exposedRoutes) Len() int      { return len(e) }
+func (e exposedRoutes) Swap(i, j int) { e[i], e[j] = e[j], e[i] }
+func (e exposedRoutes) Less(i, j int) bool {
+	a, b := e[i], e[j]
+	httpA, httpB := strings.HasPrefix(a, "https://"), strings.HasPrefix(b, "https://")
```

Comment:
Had this originally, not sure it was more readable. Should probably be
prefixA and prefixB.